templates/index.html

```diff
@@ -5,54 +5,74 @@
         <link href="https://fonts.googleapis.com/css?family=Open+Sans:400,700&display=swap" rel="stylesheet">
 {% endblock %}
 {% block body %}
+        <div class="grid-container">
 
-        {% if username %}
-        <p id="greeting">Hello {{ username }}!</p>
-        <div id="login"><a href="{{ url_for('logout') }}">Logout</a></div>
-
-        {% else %}
-        <p id="greeting">You have to log in with Wikidata to edit</p>
-        <div id="login"><a href="{{ url_for('login') }}">Login</a></div>
-        {% endif %}
-        <div id="languagesetting">
-        <label>Language</label>
-        <select id="languageselector">
-                {% for lang in languages %}
-                <option value="{{lang[1]}}" {{'selected' if currentlang == lang[1]}}>{{lang[0]}}</option>
-                {% endfor %}
-        </select>
-        </div>
+                <div id="header">
+                        <p id="info-symbol">&#9432</p>
+                        <div id="login"><a href="/machtsinn/logout">Logout</a></div>
+                        {% if username %}
+                        <p id="greeting">Hello {{ username }}!</p>
+                        <div id="login"><a href="{{ url_for('logout') }}">Logout</a></div>
```

Comment:
Here login and logout got mixed up.

Reply:
And the end for this if is missing. I think this causes the site to look that broken.